sdk/cosmos/azure-cosmos/azure/cosmos/_auth_policy.py

```diff
@@ -17,6 +17,16 @@
 
 
 class CosmosBearerTokenCredentialPolicy(BearerTokenCredentialPolicy):
+    AadDefaultScope = "https://cosmos.azure.com/.default"
+
+    def __init__(self, credential, account_scope: str, override_scope: Optional[str] = None):
+        self._account_scope = account_scope
+        self._override_scope = override_scope
```

Comment:
We don't need to store these two values right? Doesn't look like we're using them anywhere else.